crates/wasmtime/src/config.rs

```diff
@@ -1450,6 +1454,13 @@ impl Config {
         self
     }
 
+    /// This option is disabled by default.
```

Comment:
Chris mentioned a longer doc comment on the CLI flag below, but this also seems to have [prior art](https://github.com/bytecodealliance/wasmtime/blob/fe3db2a59e8b5f05d6c58c23a59799067a4dcb35/crates/wasmtime/src/config.rs#L1004) for linking to bytecodealliance.github.io for docs pages. i'm not sure if it makes sense to basically copy the CLI flag's help text here, or maybe have the flag say something a little shorter like "enables memory checking for wasm programs, see the doc comment over on config for more"...?

suppose that also depends on what you've changed through the first iteration of feedback here. i'll keep an eye out!